torch_xla/csrc/tensor_util.cpp

```diff
@@ -62,6 +62,23 @@ struct Caster<tsl::bfloat16> {
     return static_cast<D>(static_cast<float>(value));
   }
 };
+
+template <>
+struct Caster<at::Float8_e5m2> {
+  template <typename D>
+  D cast(const at::Float8_e5m2& value) const {
+    return static_cast<D>(static_cast<float>(value));
+  }
+};
+
+template <>
+struct Caster<tsl::float8_e5m2> {
+  template <typename D>
+  D cast(const tsl::float8_e5m2& value) const {
+    return static_cast<D>(static_cast<float>(value));
+  }
+};
```

Comment:
what's this `Caster ` used for?

Reply:
It's called in https://github.com/pytorch/xla/blob/master/torch_xla/csrc/tensor_util.cpp#L149C6-L149C17, implementing memcpy